Makefile

```diff
@@ -15,7 +15,7 @@ update: ## Update dependencies
 	$(DC_RUN_PHP) env XDEBUG_MODE=off composer update
 test: test-unit test-integration ## Run unit and integration tests
 test-unit: ## Run unit tests
-	$(DC_RUN_PHP) env XDEBUG_MODE=coverage vendor/bin/phpunit --testsuite unit --colors=always --coverage-text --testdox --coverage-clover coverage.clover --coverage-html=tests/coverage/html --log-junit=junit.xml
+	$(DC_RUN_PHP) env XDEBUG_MODE=coverage vendor/bin/phpunit --testsuite unit --colors=always --testdox --coverage-clover coverage.clover --coverage-html=tests/coverage/html --log-junit=junit.xml
```

Comment:
The `coverage-text` output clutters the CLI / hides the test results; `coverage-html` seems more useful overall.

Reply:
I think we originally used `coverage-text` because `coverage-html` didn't give us everything we wanted at the time.  I'm sure it's better now.